tests/chainer_tests/training_tests/extensions_tests/test_print_report.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import unittest
+
+from mock import MagicMock
+
+from chainer import testing
+from chainer.training import extensions
+
+class TestPrintReport(unittest.TestCase):
+    def _setup(self, delete_flush=False):
+        self.logreport = MagicMock(spec=extensions.LogReport(
+            ['epoch'], trigger=(1, 'iteration'), log_name=None))
+        self.stream = MagicMock()
+        if delete_flush:
+            del self.stream.flush
+        self.report = extensions.PrintReport(
+            ['epoch'], log_report=self.logreport, out=self.stream)
+
+        self.trainer = testing.get_trainer_with_mock_updater(
+            stop_trigger=(1, 'iteration'))
+        self.trainer.extend(self.logreport)
+        self.trainer.extend(self.report)
+        self.logreport.log = [{'epoch': 0}]
+
+    def test_stream_with_flush_is_flushed(self):
+        self._setup(delete_flush=False)
+        self.assertTrue(hasattr(self.stream, 'flush'))
+        self.stream.flush.assert_not_called()
+        self.report(self.trainer)
+        self.stream.flush.assert_called_once_with()
```

Comment:
Use `assert_called_with`.
Even if it is called more than once, it doesn't mean the feature is broken.